### PR TITLE
add hybrid browser and disruptor test

### PR DESCRIPTION
**Pull request by mdcruz**
Also updates the README for more instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,42 @@
...
+- Browser testing via k6 browser
+- Hybrid performance test (HTTP + xk6-disruptor, browser + xk6-disruptor)
+- and many more!
+
+This can be used for demonstrations or workshops.
+
+## Requirements
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
```
Comment (dgzlopes) [inline README.md:21]: Can we note here that the Kubernetes setup is only required if you want to test the disruptor? :eyes: 

All other k6 scripts will work with plain Docker.

Comment (mdcruz) [reply README.md]: Resolve in https://github.com/grafana/quickpizza/pull/4/commits/1fc71ef5f8e0ce58240c7ec146253e28dc55c078

Comment (dgzlopes) [reply README.md]: Sweet!

File: README.md [hunk 2]
```diff
@@ -2,26 +2,54 @@
 
 ![Screenshot from 2023-03-18 15-29-05](https://user-images.githubusercontent.com/8228060/226112255-fe2d4cdc-193e-4c23-8a36-3d8f60baaf03.png)
 
+This project contains an awesome application called `QuickPizza` and also demonstrates how to use the different features of k6. 
+
+You'll find sample tests about:
```
Comment (dgzlopes) [inline README.md:7]: Thoughts on being a bit less concrete in here? Like, saying that it covers k6's foundations, with a few words inside the parenthesis, and also X, Y, Z. With the current list, I feel bad about not including all the things that are doable :smile:

**Comment by dgzlopes** [comment]
I like the shape this is taking. Thanks, @mdcruz!

One problem with the current directory organization is that all scripts that import the example data or the shared/frontend module are broken. Also, it is hard to follow with folders inside folders.

Any thoughts on:

- /k6
  - /foundations
     - /data
     - /lib (rename of the current shared folder)
     - /extension
     - all main tests
  - /browser (specific tests / deep dive)
  - /disruptor (specific tests / deep dive)
  - /extra/misc/something (hybrid tests, WebSockets example, and more future things, etc.) - not sure about the name.

File: README.md [hunk 3]
```diff
@@ -2,26 +2,54 @@
 
 ![Screenshot from 2023-03-18 15-29-05](https://user-images.githubusercontent.com/8228060/226112255-fe2d4cdc-193e-4c23-8a36-3d8f60baaf03.png)
 
+This project contains an awesome application called `QuickPizza` and also demonstrates how to use the different features of k6.
```
Comment (dgzlopes) [inline README.md:5]: In the hackathon project, I used QuickPizza; I listed a few technical details of the app:

Screenshot from 2023-06-22 13-41-34

It would be lovely if we could provide some of that info!

**Comment by mdcruz** [comment]
@dgzlopes I've made more changes. Can you review it again, please? :) Thanks!

File: README.md (hunk 2 above)
Comment (mdcruz) [reply README.md]: Have changed this to be more general: `It also demonstrates the basic and advanced functionalities of k6, ranging from a basic load test to using different modules and extensions.`

File: README.md (hunk 3 above)
Comment (mdcruz) [reply README.md]: I've added a few details! Please look at https://github.com/grafana/quickpizza/pull/4/commits/8dc9eb68e0fdb3d1f6424de3ccd08f25113baa18#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5-R9

File: README.md [hunk 4]
```diff
@@ -2,26 +2,49 @@
 
 ![Screenshot from 2023-03-18 15-29-05](https://user-images.githubusercontent.com/8228060/226112255-fe2d4cdc-193e-4c23-8a36-3d8f60baaf03.png)
 
+## What is QuickPizza? 🍕🍕🍕
+
+`QuickPizza` is a web application, used for demonstrations and workshops, that generates new and exciting pizza combinations! 
+
+The app is built using [SvelteKit](https://kit.svelte.dev/) for the frontend and [Go](https://go.dev/) for the backend.
+
+It also demonstrates the basic and advanced functionalities of k6, ranging from a basic load test to using different modules and extensions.
```
Comment (dgzlopes) [inline README.md:11]: NIT: The app doesn't demonstrate the functionalities of k6. The tests do, and we use the app to exemplify. 

I would rephrase this, but not sure how.

File: README.md [hunk 5]
```diff
@@ -2,26 +2,49 @@
...
+Now you can go to [localhost:3333](http://localhost:3333) and get some pizza recommendations!
 
 ## Using k6 to test it
 
-Requirements:
-- Grafana k6 (v.0.43.1 or higher)
+All tests live in the `k6` folder. Within this folder, you will find the following folders:
 
-All tests live in the `k6` folder. To run them, you can use the `k6 run` command:
+- [foundation](k6/foundations/) - covers the basic functionalities of k6.
```
Comment (dgzlopes) [inline README.md:38]:
```suggestion
- [foundations](k6/foundations/) - covers the basic functionalities of k6.
```

File: README.md [hunk 6]
```diff
@@ -2,26 +2,49 @@
...
+- [browser](k6/browser/) - covers a more deep-dive look on how to use the k6 browser module for browser and web performance testing.
+- [disruptor](k6/disruptor/) - covers a more deep-dive look on how to use xk6-disruptor for failure injection testing.
+- [hybrid](k6/advanced) - covers tests that are more advanced such as hybrid tests, tracing, etc.
```
Comment (dgzlopes) [inline README.md:41]:
```suggestion
- [advanced](k6/advanced) - covers tests that are more advanced such as hybrid tests, tracing, etc.
```

File: README.md [hunk 7]
```diff
@@ -64,7 +97,19 @@ docker run -p 9090:9090 prom/prometheus --config.file=/etc/prometheus/prometheus
              --web.enable-remote-write-receiver
 ```
 
-### Deploy to kubernetes
+## Deploy application to Kubernetes
+
+When working with the xk6-disruptor test, you need to deploy the pizza application to Kubernetes.
```
Comment (dgzlopes) [inline README.md:102]:
```suggestion
If you want to run a test that uses xk6-disruptor, you need to deploy QuickPizza to Kubernetes.
```

File: README.md [hunk 8]
```diff
@@ -64,7 +97,19 @@ docker run -p 9090:9090 prom/prometheus --config.file=/etc/prometheus/prometheus
...
+
+To start, make sure you stop the Docker container first for `QuickPizza`.
```
Comment (dgzlopes) [inline README.md:104]:
```suggestion
If you are already running the app with Docker, make sure you stop the `QuickPizza` container first.
```

**Review by dgzlopes — APPROVED** [review]
I love the changes. Thanks for working on this @mdcruz!
